applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -1624,21 +1716,22 @@ int unicast_client_send(struct le_audio_encoded_audio enc_audio)
 	return 0;
 }
 
-int unicast_client_disable(void)
+int unicast_client_disable(uint8_t cig_index)
```

Comment:
Parameter unused.

Reply:
Will add a `ARG_UNUSED(cig_index)`. I added it so that the functions are symmetric